README.md

```diff
@@ -4,56 +4,142 @@
 
 Currently, the researcher or the data user has to answer a number of queries to get the desired results and it often requires iteration. 
 
-The aim of query-tol-ai would be to make this search process more user-friendly by adding an LLM style chatbot interface. This is to be done by utilizing large language models that will be able to interpret the user prompts and initiate the API calls accurately giving the user the desired results.
+The aim of query-tool-ai would be to make this search process more user-friendly by adding an LLM style chatbot interface. This is to be done by utilizing large language models that will be able to interpret the user prompts and initiate the API calls accurately giving the user the desired results.
 
-## Local installation
-- Clone the repository:
-  ```
+## Local installation - 
+  ### Clone the repository :
+  ```bash
   git clone https://github.com/neurobagel/query-tool-ai.git
   cd query-tool-ai
   ```
-- Create and activate a virtual environment:
-   ```
+  
+  Before proceeding with either you need to set the environment variables.
+  ### Set the environment variables - 
+   | Environment Variable   | Type    | Required                                 | Default Value if Not Set | Example                                                   |
+   | ---------------------- | ------- | ---------------------------------------- | ------------------------ | --------------------------------------------------------- |
+   | `HOST`                 | string  | No                                       | `0.0.0.0`                | `127.0.0.1`                                               |
+   | `PORT`                 | integer | No                                       | `8000`                   | `8080`                                                    |
+
+
+  After cloning the repository, you can choose to either use the Dockerized version or run the application locally using Python. 
```

Comment:
```suggestion
  ```
  After cloning the repository, you can choose to either use the Dockerized version or run the application locally using Python but before proceeding with either you need to set the environment variables.
  
  ### Set the environment variables - 
   | Environment Variable   | Type    | Required                                 | Default Value if Not Set | Example                                                   |
   | ---------------------- | ------- | ---------------------------------------- | ------------------------ | --------------------------------------------------------- |
   | `HOST`                 | string  | No                                       | `0.0.0.0`                | `127.0.0.1`                                               |
   | `PORT`                 | integer | No                                       | `8000`                   | `8080`                                                    | 
```

Reply:
This looks a bit weird in the GitHub editor, my suggestion is essentially to move the `After cloning the repository....` sentence before the `Set the environment variables` section